java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -369,4 +371,43 @@ CompletableFuture<Long> linsert(
      * }</pre>
      */
     CompletableFuture<Long> lpushx(String key, String[] elements);
+
+    /**
+     * Pops one or more elements from the first non-empty list <code>key</code> from the list of
+     * provided <code>key</code> names
+     *
+     * @see <a href="https://valkey.io/commands/lmpop/">valkey.io</a> for details.
+     * @param numkeys The number of provided <code>key</code>s.
```

Comment:
numkeys shouldn't be an argument - we calculate it in runtime. See `bzmpop` as an example.